pixl_dcmd/tests/test_main.py

```diff
@@ -178,12 +181,38 @@ def test_anonymise_and_validate_as_external_user() -> None:
     assert dataset != pydicom.dcmread(dataset_path)
 
 
-# TODO: test that anonymise_and_validate_dicom() works as expected
-# https://github.com/SAFEHR-data/PIXL/issues/418
+def ids_for_parameterised_test(val):
+    """Generate test ID for parameterised tests"""
+    if isinstance(val, pathlib.Path):
+        return val.stem
+    return str(val)
```

Comment:
can we have this as a shared helper function that we have already defined in conftest?

Reply:
Indeed, `ids_for_parameterised_test` is duplicated, just removed duplication and added respective import [caa9c21](https://github.com/SAFEHR-data/PIXL/pull/493/commits/caa9c211b5de4faf64ba8c88714748d6a52ff7df)